django/thunderstore/modpacks/tasks.py

```diff
@@ -0,0 +1,50 @@
+import base64
+import io
+import zipfile
+import yaml
+
+from celery import shared_task
+from typing import Dict
+from thunderstore.core.settings import CeleryQueues
+from thunderstore.modpacks.models.legacyprofile import LegacyProfile, LegacyProfileMetaData
+
+def parse_package_name_and_team(s: str):
```

Comment:
Why not use the [PackageReference](https://github.com/thunderstore-io/Thunderstore/blob/master/django/thunderstore/repository/package_reference.py#L12) class that's purpose built for parsing the package reference?